docs/advanced/document-store/benchmark.md

```diff
@@ -44,65 +43,225 @@ We are interested in the single query performance on the above tasks, which mean
+|-----------------|-------------:|-----:|----------:|--------------------:|-----------------------:|--------------:|----------:|------------:|------------:|
+|              16 |          128 |   64 |     0.960 |                1.53 |                   0.38 |        148.67 |      0.36 |       24.42 |       46.17 |
+|              16 |          128 |  128 |     0.988 |                1.72 |                   0.36 |        130.47 |      0.34 |      210.10 |      227.95 |
+|              16 |          128 |  256 |     0.996 |                1.99 |                   0.37 |        141.49 |      0.35 |      117.17 |      164.43 |
+|              16 |          256 |   64 |     0.965 |                1.54 |                   0.37 |        186.36 |      0.36 |       32.40 |       45.42 |
+|              16 |          256 |  128 |     0.990 |                1.69 |                   0.36 |        188.02 |      0.35 |       93.48 |      168.10 |
+|              16 |          256 |  256 |     0.997 |                2.07 |                   0.36 |        183.66 |      0.36 |       18.86 |       35.82 |
+|              32 |          128 |   64 |     0.975 |                1.58 |                   0.40 |        156.41 |      0.34 |       16.17 |       31.42 |
+|              32 |          128 |  128 |     0.993 |                1.81 |                   0.37 |        147.05 |      0.35 |       19.81 |       39.87 |
+|              32 |          128 |  256 |     0.998 |                2.15 |                   0.38 |        144.64 |      0.34 |       29.62 |       40.21 |
+|              32 |          256 |   64 |     0.984 |                1.62 |                   0.37 |        211.81 |      0.35 |       32.65 |       35.15 |
+|              32 |          256 |  128 |     0.996 |                1.84 |                   0.37 |        210.85 |      0.35 |      141.31 |      175.80 |
+|              32 |          256 |  256 |     0.999 |                2.25 |                   0.37 |        204.65 |      0.35 |       22.13 |       31.54 |
 
+````
 
-````{tab} Default HNSW parameters
+````{tab} Qdrant
+
+| max connections | ef construct |  ef  | Recall@10 | Find by vector (ms) | Find by condition (ms) | Create 1M (s) | Read (ms) | Update (ms) | Delete (ms) |
+|-----------------|-------------:|-----:|----------:|--------------------:|-----------------------:|--------------:|----------:|------------:|------------:|
+|              16 |           64 |   32 |     0.968 |               47.80 |                  44.96 |       5889.66 |      5.70 |       46.11 |       47.24 |
+|              16 |           64 |   64 |     0.990 |               47.88 |                  44.72 |       5892.35 |      2.72 |       46.61 |       47.30 |
+|              16 |           64 |  128 |     0.997 |               47.80 |                  44.95 |       5881.15 |      2.84 |       45.34 |       47.36 |
```

Comment:
higher recall as the ef value increase implies that the configuration is correctly applied, although find by vector stays constant.
I doubt big network overhead here, I'm trying to confirm